src/celeritas/ext/detail/GeantProcessImporter.cc

```diff
@@ -438,17 +445,17 @@ ImportPhysics2DVector import_physics_2dvector(G4Physics2DVector const& g4pv,
     double const v_scaling = native_value_from_clhep(units[2]);
 
     ImportPhysics2DVector pv;
-    pv.x.resize(g4pv.GetLengthX());
-    pv.y.resize(g4pv.GetLengthY());
+    pv.x.resize(g4pv.GetLengthY());
+    pv.y.resize(g4pv.GetLengthX());
     pv.value.resize(pv.x.size() * pv.y.size());
 
     for (auto i : range(pv.x.size()))
     {
-        pv.x[i] = g4pv.GetX(i) * x_scaling;
+        pv.x[i] = g4pv.GetY(i) * y_scaling;
         for (auto j : range(pv.y.size()))
         {
-            pv.y[j] = g4pv.GetY(j) * y_scaling;
-            pv.value[pv.y.size() * i + j] = g4pv.GetValue(i, j) * v_scaling;
+            pv.y[j] = g4pv.GetX(j) * x_scaling;
+            pv.value[pv.y.size() * i + j] = g4pv.GetValue(j, i) * v_scaling;
```

Comment:
Remind me later to improve the `corecel/data/HyperslabIndexer.hh` class so we can use it for consistent ordering for multi-D data in Celeritas...
```suggestion
    HyperslabIndexer index{{pv.x.size(), pv.y.size()}};
    pv.value[index(i, j)] = g4pv.GetValue(j, i) * v_scaling;
```